linux/scripts/pantos-service-node-celery

```diff
@@ -34,8 +34,14 @@ while [ $# -gt 0 ]; do
       ;;
     -s|--status)
       echo "Querying the status of the celery worker"
-      exec ./bin/python -m celery -A pantos.servicenode report 
-      exit $?
+      exec ./bin/python -m celery -A pantos.servicenode report
+      status=$?
+      if [ $status -ne 0 ]; then
```

Comment:
What health status are you trying to get here with celery? By running `celery -A pantos.servicenode` you start a new worker which dumps a report, but that does not tell you if another worker is up and healthy.

Is that intended?

Reply:
Good question, we did have this already before so I pretty much kept it. We just want to print a message to know what the status is, but if there's another command we should use please let me know 🙏

Reply:
What status? That the worker is able to connect?

If you want to have the status of other workers, I believe that the bugticket (https://bitpanda.atlassian.net/browse/PAN-2067) will give us the answer to what command to be used. 

In order to unblock you, we can create a ticket, which is linked to PAN-2067, which will change it here if required and then I will approve it. WDYT?